docs/installation.md

```diff
@@ -203,3 +182,30 @@ They are
     ```
 
 Consult the AGNI wiki if you encouter issues.
+
+### Chemical kinetics atmosphere model (**VULCAN**)
+
+1. Clone the model
+
+    ```console
+    git clone git@github.com:exoclime/VULCAN.git
+    cd VULCAN
+    ```
+2. Compile the FastChem extension
+
+    ```console
+    cd fastchem_vulcan
+    ```
+
+    On MacOS you will need to edit `make.globaloptions` to reflect  a GNU-compatible `g++` executable, not the Apple one (see
+     [Troubleshooting](./troubleshooting.md) if the next step results in an error.
+
+    ```console
+    make
+    cd ../../
+    ```
+3. Install Python dependencies
+
+    ```console
+    pip install sympy
+    ```
```

Comment:
Although I understand why this is needed, it would be better if Vulcan would declare its own dependencies, either through an `requirements.txt` or `pyproject.toml`. May be worth to contribute a patch to that code, so that it can be installed like mors or janus (e.g. `pip install vulcan/.`). That also avoids PROTEUS having to set the PYTHONPATH for Vulcan in `proteus.env`.